configure.ac

```diff
@@ -46,7 +46,6 @@ fi
 AC_PROG_CPP
 AC_PROG_MAKE_SET
 AC_PROG_INSTALL
-AC_PROG_MKDIR_P
```

Comment:
You're right. Forget about this part, I misinterpreted a warning, the obsolete macro use is in a gettext m4 file. Can I rebase and remove this patch and force-push, or is there a better way to get rid of this?